### ISSUE TITLE
Always open in new window

### ISSUE THREAD

**Issue by ryanmeisters** [ryanmeisters/Xcode-Plugin-Open-Sublime-Text#8]
Open in new sublime text window instead of opening as a new tab in any currently open windows. 

All we need to do is add --new-window to the subl command execution

**Comment by pasta12** [comment]
I'm curious why you would want this? I'd think you'd want to use what the user has their `open_files_in_new_window` preference set to.

**Comment by ryanmeisters** [comment]
Hm. Interesting point. 

Lately I'm using sublime text for miscellaneous notes and things, so I always have a number of windows open. When I open a file from inside Xcode, it opens in an existing sublime window, even if the existing window is several desktops away, which is kind of annoying/distracting. 

What is your workflow like with the Open In Sublime Text plugin? Mine is something like this: 

Working in xcode => open in sublime => make edit that is faster in sublime => close sublime (command-w)

Also, the NSTask that launches the sublime executable is launched on a background thread and waits until sublime exits (i.e. the opened file is closed) and then focus is restored to Xcode.

**Comment by pasta12** [comment]
Ah, you make a good point about switching desktops. That would be very annoying.

My current workflow is nothing because I'm using Sublime Text 2 so Xcode crashes. I'm sure you've seen my pull request for that though. I could add this change to that PR if you'd like as well. Easy change. I would add it in the way your originally described.

**Comment by ryanmeisters** [comment]
That would be awesome. Let's do it as a separate pull request though I think.

**Comment by pasta12** [comment]
Sounds good. I will be doing this over the weekend.

**Comment by pasta12** [comment]
Do you want the PR to the develop or master branch?

**Comment by ryanmeisters** [comment]
Develop please :)

**Comment by pasta12** [comment]
Can you get your develop branch up to date? I'd like to pull from upstream but your history is different than mine. Trying to avoid merge commits as much as possible. Theoretically I was expecting your develop history to look the same as your master branch at this point.

**Comment by ryanmeisters** [comment]
I actually prefer to follow the branching model outlined [here](http://nvie.com/posts/a-successful-git-branching-model/). I'm not a big fan of fast-forward merges. I think it's useful to maintain the branch history, and I think it's important to only have release commits on the master branch. 

I think our histories are different because your previous work branched from master instead of develop, so I had to rebase it onto develop. The easiest path forward might just be to clone a new copy of the repository and always branch from develop in the future :) Sorry about that!

**Comment by pasta12** [comment]
For my previous PR, I rebased master into develop because develop was behind, then branched off of develop, did my work, rebased it into develop, and made my PR from my develop to yours.

I'll take the approach at that link for future PRs. I'm going to have to take some time to read through it again, but I have read it in the past. That being said, your develop branch history still doesn't make sense to me :) I'll get it set up the way you want it and make a PR. Possibly not until tomorrow.

**Comment by pasta12** [comment]
Will the release be 1.3?

**Comment by ryanmeisters** [comment]
Hm. I see. Sorry about that. Maybe I did forget to push the dev branch 😝. What doesn't make sense about the history?

It will be 1.3, but I will take car of bumping the version and merging into master
